packages/components/organisms/f-content-cards/src/components/ContentCards.js

```diff
@@ -208,41 +208,39 @@ export default {
          * @return {Promise<void>}
```

Comment:
This is no longer correct, I think?
```suggestion
         * @return {void}
```
We can probably remove the `await` in the `mounted()` hook, as well.